qgis-app/settings_docker.py

```diff
@@ -159,4 +159,17 @@
 MATOMO_URL="//matomo.qgis.org/"
 
 # Default primary key type
-DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
\ No newline at end of file
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
+
+
+# Sentry
+SENTRY_DSN = os.environ.get("SENTRY_DSN", "")
+
```

Comment:
@Xpirix perhaps add a check here, if sentry_dsn not specified then we don't need to init the sentry

Reply:
@dimasciput Sure, I've added it.